demo/app.py

```diff
@@ -203,6 +203,7 @@ def update_observations(n_clicks, form_values, table_data):
                 "margin-left": "260px",
                 "padding": "20px",
                 "padding-left": "30px",
+                "padding-bottom": "70px",
```

Comment:
I have added a "padding-bottom" now to prevent the footer blocking the main component.